src/main/java/uk/gov/hmcts/reform/sandl/snlapi/Application.java

```diff
@@ -4,10 +4,14 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
 import org.springframework.cloud.netflix.hystrix.dashboard.EnableHystrixDashboard;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
 @SpringBootApplication
 @EnableCircuitBreaker
 @EnableHystrixDashboard
+@EnableWebMvc
+@ComponentScan
```

Comment:
@SpringBootApplication already has ComponentScan, redundant not needed
https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-using-springbootapplication-annotation.html

Reply:
If the test does not work then add to the test @WebMvcTest(RootController.class)